DOCS/man/options.rst

```diff
@@ -5277,7 +5277,8 @@ them.
     ``ewa_lanczossharp``
         A slightly sharpened version of ewa_lanczos, preconfigured to use an
         ideal radius and parameter. If your hardware can run it, this is
```

Comment:
> If your hardware can run it...

This line could be removed if it's added to to `gpu-hq` preset. Although, I think using polar scalers for `cscale` is just a big overkill.